src/styles/themes/_default-theme.scss

```diff
@@ -181,7 +181,7 @@
         Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
     --font-size: 16px;
 
-    /* ------ Tabs theme ------ */
+    // ------ Tabs theme ------
```

Comment:
`/* */` style comments remain in css output
`//` style comments get dropped in output